ironfish-cli/src/commands/wallet/transactions/index.ts

```diff
@@ -62,6 +62,14 @@ export class TransactionsCommand extends IronfishCommand {
       options: ['notes', 'transactions', 'transfers'],
       helpGroup: 'OUTPUT',
     }),
+    'filter.start': Flags.string({
+      description: 'include transactions after this date (inclusive). Example: 04/20/2023',
+      parse: (input) => Promise.resolve(new Date(input).toISOString()),
+    }),
+    'filter.end': Flags.string({
```

Comment:
nit - should the names for these be more date-specific? `start` and `end` could also refer to sequences. I suppose we could handle that if/when we ever add filtering on sequence

Reply:
Suggestion: before/after